generators/cucumber/templates/env.rb

```diff
@@ -37,4 +37,32 @@
     log.info "SAUCE SERVER LOG URL: #{Capybara::Settings.sauce_resource_path('selenium-server.log')}"
     Capybara::Settings.update_sauce_job_status(passed: DataStorage.extract('sauce', :status))
   end
+end
+
+# BrowserStack settings
+
+url = settings.bs_url
+
+capabilities = Selenium::WebDriver::Remote::Capabilities.new
```

Comment:
It is wrong place for this code with settings, because we need to share it with RSpec as well. See how it implemented for Testingbot
https://github.com/strongqa/howitzer/blob/develop/lib/howitzer/capybara/settings.rb#L156